cds/modules/records/permissions.py

```diff
@@ -311,6 +318,17 @@ def has_read_record_permission(user, record):
     return has_admin_permission()
 
 
+def has_read_record_eos_path_permission(user, record):
+    """Check if user has eos path permissions."""
+    user_provides = get_user_provides()
+    # Allow e-group members only
+    read_access_groups = current_app.config.get('VIDEOS_EOS_PATH_EGROUPS', [])
+
+    if not set(user_provides).isdisjoint(set(read_access_groups)):
```

Comment:
nice! is it worth to add a quick comment to explain?

Reply:
You can eventually add unit tests for this, it will also help to explain...

Reply:
done, thanks!